.github/workflows/deploy.yml

```diff
@@ -90,6 +90,74 @@ jobs:
       - name: Run Deployment Script Previous Major Version
         if: ${{ github.ref != 'refs/heads/main' }}
         run: script/deploy-canary-demo -prev
+  deploy_ChimeSDKMeetings_ChimeSDKMediaPipelines_beta_demo:
+    name: Beta - ChimeSDKMeetings Client/ChimeSDKMediaPipelines Client - Deploy the Serverless Meeting Demos
```

Comment:
Hope this `name` field is fine to accept `/`. Let me know if you did a test run and this is Ok. I could not find any constraints on the GitHub action job's name field.

Reply:
removed the `/` to be cautious